manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -67,6 +67,7 @@ defaults:
   podAnnotations: {}
   serviceAnnotations: {}
   serviceAccountAnnotations: {}
+  sidecarInjectorWebhookAnnotations: {}
```

Comment:
in cause someone want add more things, what about change this to
```
  sidecarInjectorWebhook:
    annotations: {}
```

cc @istio/wg-environments-maintainers WDYT?

Reply:
`sidecarInjectorWebhook` is already defined at [values.yaml](https://github.com/istio/istio/blob/master/manifests/charts/istio-control/istio-discovery/values.yaml#L107). Does it make more sense to move annotations under that?

Reply:
sgtm

Reply:
I attempted to move this to `Values.sidecarInjectorWebhook.annotation`, but this resulted in annotation being added to `istiod-injector-configmap.yaml`.

```yaml
# Source: istio-istiod/templates/istiod-injector-configmap.yaml
apiVersion: v1
kind: ConfigMap
<SNIPPED>
data:
  values: |-
    {
      "global": {
        "autoscalingv2API": true,
 <SNIPPED>
      "sidecarInjectorWebhook": {
        "alwaysInjectSelector": [],
        "annotations": {
          "argocd.argoproj.io/sync-wave": "test"
        },
        "defaultTemplates": [],
        "enableNamespacesByDefault": false,
 <SNIPPED> 
```

I believe the correct location should be `Values.sidecarInjectorWebhookAnnotation` itself because:

1. It aligns with similar fields like `podAnnotations` and `serviceAnnotations`.
2. `Values.sidecarInjectorWebhook` appears to configure the injected sidecar, not the webhook itself.

Thoughts?

Reply:
In that case, can we just do `Values.webhookAnnotations`?